x-pack/test_serverless/api_integration/test_suites/common/platform_security/views.ts

```diff
@@ -6,131 +6,107 @@
  */
 
 import expect from 'expect';
-import { InternalRequestHeader, RoleCredentials } from '../../../../shared/services';
+import { SupertestWithRoleScopeType } from '@kbn/test-suites-xpack/api_integration/deployment_agnostic/services';
 import { FtrProviderContext } from '../../../ftr_provider_context';
 
 export default function ({ getService }: FtrProviderContext) {
   const svlCommonApi = getService('svlCommonApi');
-  const svlUserManager = getService('svlUserManager');
-  const supertestWithoutAuth = getService('supertestWithoutAuth');
-  let roleAuthc: RoleCredentials;
-  let internalReqHeader: InternalRequestHeader;
+  const roleScopedSupertest = getService('roleScopedSupertest');
+  let supertestAdminWithCookieCredentials: SupertestWithRoleScopeType;
+  let supertestAdminWithApiKey: SupertestWithRoleScopeType;
 
   describe('security/views', function () {
     before(async () => {
-      roleAuthc = await svlUserManager.createM2mApiKeyWithRoleScope('admin');
-      internalReqHeader = svlCommonApi.getInternalRequestHeader();
+      supertestAdminWithCookieCredentials = await roleScopedSupertest.getSupertestWithRoleScope(
+        'admin',
+        {
+          useCookieHeader: true,
+          withInternalHeaders: true,
+        }
+      );
+      supertestAdminWithApiKey = await roleScopedSupertest.getSupertestWithRoleScope('admin', {
+        withInternalHeaders: true,
+      });
     });
     after(async () => {
-      await svlUserManager.invalidateM2mApiKeyWithRoleScope(roleAuthc);
+      await supertestAdminWithApiKey.destroy();
     });
     describe('route access', () => {
       describe('disabled', () => {
         // ToDo: unskip these when we disable login routes
         xit('login', async () => {
-          const { body, status } = await supertestWithoutAuth
-            .get('/login')
-            .set(svlCommonApi.getInternalRequestHeader())
-            .set(roleAuthc.apiKeyHeader);
+          const { body, status } = await supertestAdminWithApiKey.get('/login');
```

Comment:
These endpoints are all meant to be used without API keys (even though that's how we were testing before). We can just use cookies for all of these. Can change it here if it's quick enough, or leave it for us to do in a follow-up.